src/osd/scrubber/ScrubStore.cc

```diff
@@ -15,60 +15,62 @@ using std::vector;
 using ceph::bufferlist;
 
 namespace {
+
+/// the 'to_str()' representation of an hobject, if created with the specific
+/// parameters, prefixed with 'prefix'.
+std::string virtual_hobject_w_prefix(std::string_view prefix,
+                                     const std::string& key, snapid_t snap,
```

Comment:
nit: not sure it's worth to save some new lines by combining the parameters.

Reply:
Agree. Will fix.